python/cuda_cccl/pyproject.toml

```
@@ -21,22 +21,20 @@ dependencies = [
   "numpy",
   "cuda-python==12.9.0",
   "cupy-cuda12x",
+  "numba-cuda",
```

btw I think this will work starting today/tomorrow once we cut a release in numba-cuda:
```suggestion
  "numba-cuda[cu12]",
```